radio/src/targets/horus/hal.h

```diff
@@ -477,6 +477,49 @@
   #define AUX2_SERIAL_RCC_APB2Periph           0
 #endif
 
+//OW
+#if (defined(RADIO_T16) || defined(RADIO_T18)) && defined(AUX_SERIAL)
```

Comment:
This block of changes if most probably not needed. The definitions are already present for AUX & AUX2 serial ports, which is the only thing you seem to need. Please remove.